### DISCUSSION TITLE
Provide a REST API example in examples/

### DISCUSSION THREAD

**Discussion by PolpOnline** [maxcountryman/axum-login#252] [Ideas]
It would be great to have an authenticated REST API example in the `examples/` directory, because not everybody wants to start with a template engine such as `askama`.

Even better if with another DB, such as Postgres (I know switching from SQLite to another database on sqlx is trivial, it would just be _useful_ to have a template like that)

**Comment by maxcountryman** [comment]
An API example is a fine idea provided it's minimal and doesn't increase the maintenance burden by much. You're welcome to propose an example in a PR.

Others have provided examples and templates that cover Postgres; ultimately separate repos are probably the right place for more involved templates and I intentionally want to minimize the maintenance burden of examples in this repo.
